### DISCUSSION TITLE
Can we get achieve?

### DISCUSSION THREAD

**Discussion by youtonit** [community/community#18818] [Other Feature Feedback, Questions, & Ideas]
Can we get achieve?

**Comment by kotleni** [comment]
It's impossible.

**Reply by lhh1171** [reply]
Of course,I got an achievement named Pull Shark

**Reply by Oussama-Fajraoui** [reply]
achievement Pull Shark is so easy 
merging with another repo will increment it

**Reply by cardison** [reply]
OK COURSE  you can get achieve it is so easy all you want to understand what are all the achievements about.
https://github.com/Schweinepriester/github-profile-achievements  click this link to know about how to get all achievements

**Comment by melvincwng** [comment]
hi @youtonit, I assume you are referring to achievements.

If that's the case, please refer to this link for a list of achievements available on Github - https://github.com/Schweinepriester/github-profile-achievements

**Comment by alamin-karno** [comment]
Yes, you can get achievements. 

You have to do certain things to get those achievements on your profile.

Follow this link for more details about how to ger achievements:

[GitHub Profile Achievements 🏆](https://github.com/Schweinepriester/github-profile-achievements)

**Comment by Tran-Quyen** [comment]
You can go to this link [GitHub Profile Achievements ](https://github.com/Schweinepriester/github-profile-achievements) to understand what to do

**Comment by ApidriuC** [comment]
You can find more information about the achievements here:

https://github.com/Schweinepriester/github-profile-achievements

**Comment by Andreaa-Dev** [comment]
You can easily get the Yolo by merging your own branch. I tried and worked 👍
